ops/terraform/modules/aou-rw-reporting/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable credentials_file {
```

Comment:
Values for these can be spcified on the command line `with -var-file` or `-var`, provided at runtime interactively. Our `tfvars` files will live in `workbench-devops`.